### PR TITLE
Specify node version (nvm)

### PR DESCRIPTION
**Pull request by ddmills**
Specify the version of node using `.nvmrc` file.

[NVM](https://github.com/nvm-sh/nvm) is the most popular node version manager tool. By adding an `.nvmrc` file, anyone using nvm can quickly identify and switch to the correct version of node for this project

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+14.15.5
```
Comment (ooflorent) [inline .nvmrc:1]:
```suggestion
15.8.0
```

**Review by ooflorent — COMMENTED** [review]
That's a good idea! However, I would prefer that we use the latest node because it ships with the most up-to-date v8. This way the performance results should reflect a web user would experience.

**Comment by ddmills** [comment]
The odd numbers are experimental branches and even numbers are LTS, so 14 is LTS. But as this is a benchmark i agree that having the most up to date v8 makes a lot of sense 👍

**Comment by ooflorent** [comment]
Thanks!

**Pull request merged**